client/src/components/detailedInfo.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Row from 'react-bootstrap/Row';
+import PropTypes from 'prop-types';
+
+const DetailedInfo = ({ product }) => {
+  const slogan = product && product.slogan;
+  const description = product && product.description;
```

Comment:
Is this for conditional rendering? Seems cool, would love to know what you're doing here

Reply:
product is undefined on the first render so if I do product and product.slogan it'll give me product=undefined product.slogan= slogan